extensions/functions_rounding.yaml

```diff
@@ -0,0 +1,27 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "ceil"
+    description: "Rounding to the ceiling of the value"
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: fp32
+        return: i64
```

Comment:
The range of even an fp32 is MUCH larger than the range of i64. ceil and floor normally return floating point numbers for a reason. If you want an i64, the pattern is `cast<i64>(floor(x))`.